superbench/benchmarks/micro_benchmarks/ib_validation_performance.py

```diff
@@ -43,6 +44,13 @@ def add_parser_arguments(self):
             required=False,
             help='The IB device, e.g., mlx5_0, mlx5_$LOCAL_RANK, mlx5_$((LOCAL_RANK/2)), etc.',
         )
+        self._parser.add_argument(
+            '--set_ib_devices',
+            action='store_true',
+            default=False,
```

Comment:
or maybe change the type of this argument to `string` or `nargs="*"` with `default=None`
* default is `None`, keep previous behavior
* if set to `""` or `[]`, auto detect
* if set to `dev1,dev2` or `[dev1, dev2]`, use directly as the current `IB_DEVICES` env